docs/resources/aws_cloudfront_distribution.md

```diff
@@ -47,6 +47,8 @@ For additional information, see the [AWS API reference for CloudFront distributi
 |custom\_origin\_ssl\_protocols              | An array containing SSL/TLS protocols allowed by custom origins in this distribution. Empty if there are no custom origins (one or more standard S3 bucket origins). Current valid values are `SSLv3`, `TLSv1`, `TLSv1.1`, `TLSv1.2`. |
 |viewer\_certificate\_minimum\_ssl\_protocol | The minimum SSL/TLS protocol version in the Viewer Certificate. Current valid values: `SSLv3`, `TLSv1`, `TLSv1_2016`, `TLSv1.1_2016`, `TLSv1.2_2018`, `TLSv1.2_2019`, `TLSv1.2_2021`. |
 |s3\_origin\_config                          | `True`: if there are any S3 origin configs in the distribution (i.e. standard S3 bucket origins), else `False`. |
+|origin\_domain\_name                        | Provide the `origin_domain_name` in case want to validate `s3_origin_path`
```

Comment:
Is the `origin_domain_name` one of the properties, or is it just a parameter?

Reply:
it is an optional parameter. for this change property is `s3_origin_path`